### DISCUSSION TITLE
Transparent background for GameWidget?

### DISCUSSION THREAD

**Discussion by ahndwon** [flame-engine/flame#2124] [Q&A] [answered]
Hello, I want to overlay GameWidget over Flutter Widgets with transparent background.
But when I set the background of GameWidget to transparent, it show the background in black.
Is there a way to make it transparent?

**Comment by ufrshubham** [comment]
> But when I set the background of GameWidget to transparent, it show the background in black.

Can show how exactly are you setting background to black for GameWidget?

**Reply by ahndwon** [reply]
I was drawing color on the canvas as below.
Solved it by overriding backgroundColor method.
Thanks for the resposne!
```
class CustomGame extends Game {
  @override
  void render(Canvas canvas) {
    canvas.drawColor(Colors.transparent, BlendMode.src);
  }

  @override
  void update(double dt) {}
}
```

**Comment by erickzanardo** [comment, marked as answer]
GameWidget does not have a background color.

The black background you are seeing is most probably the color of your game which is black by default.

Try overriding the backgroundColor method of your game class to return a transparent color

**Reply by ahndwon** [reply]
Thanks!!
I was just drawing color to the canvas!
Overriding the backgroundColor method works.
